### ISSUE TITLE
Cancel/dismiss jobs UI alert

### ISSUE THREAD

**Issue by grallewellyn** [MAAP-Project/ZenHub#963]
Sujen said 
Right now, if you cancel a submitted job that is in the queued state, it deletes from your list of jobs
If you cancel a submitted job that is in the started state, it stays in your list of jobs in the job-revoked state

But I am seeing the behavior that a job stopped in the queued state is still appearing in your jobs list after you cancel, just in the job-revoked state. @marjo-luc can you please clarify?

There should be a popup to the user explaining this behavior before they start a job

Sujen suggested a tooltip, but that would be very wordy for a tooltip

**Comment by marjo-luc** [comment]
I don't think we should have a popup or a tooltip. If we need one to explain the UI, then the UI is what needs to be fixed.

That said, this is a DPS issue/behavior. 

We touched on this during our last hackathon. Jobs in a queued state that are being revoked, yet remain in the jobs list after the fact, were picked up by a worker before the revoke job went through and so were really jobs that had started.

**Comment by rtapella** [comment]
I agree with Marjorie, I don’t think we need to explain this before starting a job. We should be able to have some contextual help in the status area that explains what each status means.

I am also not sure that this is a high priority issue to fix right now.. was there a scientist confused by this?

**Comment by grallewellyn** [comment]
That could also be an option because I think the case scenario would be that a user is confused why sometimes their cancelled job is removed from the jobs list and sometimes it stays in the revoked state. So they click on the job status information button and look at the explanation for the revoked state
But we might want the popup anyway to warn users they are about to do a destructive action? 
Also there was not a scientist confused by this, I was just testing in DIT and noticed it

**Comment by marjo-luc** [comment]
I think we should start by describing the behavior in the docs. We can see how users react to the UI and adjust from there. My initial concern regarding adding a popup is that it would unnecessarily impede users. Canceling jobs is not a destructive action because queued jobs have produced no science data and started jobs aren't deleted. If we were deleting science data, then could see the need for it.

**Comment by grallewellyn** [comment]
Do we want to have the stop option for completed/ failed/ revoked jobs? I know cancelling a completed job will fail, but should users be able to see the cancel button for completed jobs? I can make a new ticket for this if people agree 
<img width="1073" alt="Screenshot 2024-04-08 at 3 06 41 PM" src="https://github.com/MAAP-Project/Community/assets/114033465/9b46bd3b-4068-4661-b3d6-0243d12fb14f">

**Comment by marjo-luc** [comment]
This was deliberate. The action menu will contain the retry feature once that's implemented, among other job-specific actions. I want to keep the options available consistent so it doesn't impeded usage. We could disable the stop button for inactive jobs if it confuses users.
